### PR TITLE
Fixing the inconsistency btw #2012 and #2066

### PR DESCRIPTION
**Pull request by mkravchik**
There was an inconsistency between two recent PRs #2012 and #2066. This one fixes it.

### CODE REVIEW

File: fuzzers/forkserver_libafl_cc/src/main.rs [hunk 1]
```diff
@@ -86,7 +86,7 @@ struct Opt {
 
 #[allow(clippy::similar_names)]
 pub fn main() {
-    const MAP_SIZE: usize = EDGES_MAP_SIZE; //65536;
+    const MAP_SIZE: usize = EDGES_MAP_SIZE_IN_USE; //65536;
```
Comment (domenukk) [inline fuzzers/forkserver_libafl_cc/src/main.rs:89]: Do we not support variable map sizes in libafl_cc forkserver yet? 
AFL++ collision free coverage at least can do that

Comment (tokatoka) [reply fuzzers/forkserver_libafl_cc/src/main.rs:89]: we do support but it is up to the harness if it uses it.

Comment (mkravchik) [reply fuzzers/forkserver_libafl_cc/src/main.rs:89]: When I try to set it to the 64K, it fails with:
thread 'main' panicked at /home/user/LibAFL/libafl/src/executors/forkserver.rs:765:13:
assertion failed: self.map_size.is_none() || map_size as usize <= self.map_size.unwrap()

I'm not sure why is this logic there, so I did not want to mess with it

Comment (tokatoka) [reply fuzzers/forkserver_libafl_cc/src/main.rs:89]: i fixed it in another PR already.

**Pull request merged**